samples/python/winml_tutorial/pywinrt_output/pyrt/src/py.Windows.Devices.Geolocation.Geofencing.h

```diff
@@ -0,0 +1,56 @@
+// WARNING: Please don't edit this file. It was generated by Python/WinRT
+
+#pragma once
+
+#include "pybase.h"
+
+#if __has_include("py.Windows.Devices.Geolocation.h")
```

Comment:
curious about the use of __has_include - is it only to light up related behavior? if so, shouldn't that be the developers decision (whether to incur the compile time, etc)?

Reply:
Please comment on #217